CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ For information on changes in released versions of Teku, see the [releases page]
 - Switched executor queue size metrics to use labelled gauge.
 - New console message when Teku switches forks. 
 - Reduce CPU usage by using batching signature verification service for aggregate attestation and sync committee contributions.
-
+- Support to `/eth/v1/keystores` endpoint to list currently validating keys.
```

Comment:
experimental and behind a dev flag, so should leave it out of changelog for now...